pkg/server/sandbox_run.go

```diff
@@ -157,6 +158,18 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 		return nil, errors.Wrap(err, "failed to generate sandbox container spec")
 	}
 	log.G(ctx).Debugf("Sandbox container %q spec: %#+v", id, spew.NewFormatter(spec))
+	sandbox.ProcessLabel = spec.Process.SelinuxLabel
+	defer func() {
+		if retErr != nil {
+			_ = label.ReleaseLabel(sandbox.ProcessLabel)
```

Comment:
nit: this one is deprecated. We can use `selinux.ReleaseLabel` which returns nothing, so we don't have to write `_ = X` everywhere.

Reply:
I'll add this for the next around of updates.

Reply:
i went ahead and updated this